package.json

```diff
@@ -36,6 +36,7 @@
     "@zero-tech/zui": "^0.24.4",
     "audio-react-recorder-fixed": "^1.0.3",
     "classnames": "^2.3.1",
+    "decimal.js": "^10.4.3",
```

Comment:
Wasn't sure if it was heavy handed to import a decimal library here, but I figured it's a tiny library and we should really use it wherever we have a wei amount (YAGNI I know)

Reply:
the only thing is, i just don't think we should ever need to use decimals. decimals should only ever be appearing in the final display string. everything else should just be integers represented as strings, or if we're doing calculations, BigNumbers. even for USD, or other tokens with minimal decimal places, we should always just be using integers.